### PR TITLE
Small update to Contributing.md

### PR DESCRIPTION
**Pull request by BramGruneir**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,8 +25,12 @@ downloaded and installed which can be somewhat time consuming. Be patient.
 Note that if you edit a `.proto` file you will need to manually regenerate the associated
 `.pb.{go,cc,h}` files using `go generate`.
 
-To add or update a dependency, use `go get -u` and then
-`glock save github.com/cockroachdb/cockroach` and commit the changes to the GLOCKFILE.
+To add or update a dependency:
+- `go get -u` to updates the dependancies or `go get {package}` to add a new dependancy
```
Comment (tschottdorf) [inline CONTRIBUTING.md]: s/updates/update
s/the dependancies//g
s/dependancy/dependency/g (also below)
s/new//

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,8 +25,12 @@ downloaded and installed which can be somewhat time consuming. Be patient.
...
+- be sure to reference it somewhere in your code
```
Comment (tschottdorf) [inline CONTRIBUTING.md]: i would leave that out. you're using it, or you don't want to add it.

**Comment by tschottdorf** [comment]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (BramGruneir) [inline CONTRIBUTING.md]: Wow.  I really messed with the english language there.  Clearly didn't proofread.  Fixing.

**Pull request merged**